app/templates/_base_page.html

```diff
@@ -1,26 +1,59 @@
-{% extends "toolkit/layouts/_base_page.html" %}
+{% extends "govuk-frontend/template.njk" %}
 
-{# Import GOV.UK Frontend components for globale usage#}
-{% from "components/breadcrumbs/macro.njk" import govukBreadcrumbs %}
-{% from "components/button/macro.njk" import govukButton %}
+{# Import GOV.UK Frontend components for globale usage #}
+{% from "govuk-frontend/components/breadcrumbs/macro.njk" import govukBreadcrumbs %}
+{% from "govuk-frontend/components/button/macro.njk" import govukButton %}
+{% from "govuk-frontend/components/error-summary/macro.njk" import govukErrorSummary %}
+{% from "govuk-frontend/components/input/macro.njk" import govukInput %}
+{% from "govuk-frontend/components/phase-banner/macro.njk" import govukPhaseBanner %}
 
-{# TODO: Remove once we start removing govuk_template, GOV.UK Frontend template uses pageTitle #}
-{% block page_title %}{% block pageTitle %}{% endblock %}{% endblock %}
+{# Import DM components #}
+{% from "digitalmarketplace/components/header/macro.njk" import dmHeader %}
+{% from "digitalmarketplace/components/footer/macro.njk" import dmFooter %}
+
+{% set assetPath = '/user/static' %}
```

Comment:
I think this line might be the cause of the assets issue

Reply:
It certainly covered it up (as my local env was happily using the User FE assets). Now fixed.